lib/screenplay_web/controllers/config_controller.ex

```diff
@@ -38,4 +42,46 @@ defmodule ScreenplayWeb.ConfigController do
         send_resp(conn, 400, "S3 Operation Failed: Put")
     end
   end
+
+  def existing_screens(conn, %{"place_ids" => place_ids, "app_id" => app_id}) do
+    app_id_atom = String.to_existing_atom(app_id)
+
+    places_and_screens =
+      place_ids
+      |> String.split(",")
+      |> Enum.map(fn place_id ->
+        filter_fn = fn
+          {_, %Screen{app_id: ^app_id_atom} = config} ->
+            place_id_has_screen?(place_id, String.to_existing_atom(app_id), config)
+
+          _ ->
+            false
+        end
+
+        live_screens =
+          ScreensConfigCache.screens(&filter_fn.(&1))
+          |> Enum.map(fn screen ->
+            Map.put(screen, :is_live, true)
+          end)
+
+        pending_screens = PendingScreensConfigCache.screens(&filter_fn.(&1))
```

Comment:
```suggestion
        pending_screens = PendingScreensConfigCache.screens(filter_fn)
```
(The syntax rules around named vs anonymous functions are weird, huh)